script/publish-docs

```diff
@@ -27,23 +28,23 @@ cd tmp/website
 git pull origin master
 
 # Delete previously generated docs for this version
-rm -rf api/$VERSION
+rm -rf "api/$VERSION"
 
 # Copy over the new docs
-mv ../../out api/$VERSION
+mv ../../out "api/$VERSION"
 
 # Update the /api/latest link to point to this version
-ln -sfn $VERSION api/latest
+ln -sfn "$VERSION" api/latest
 
 # Update the submodule
 git submodule update --init
 cd _submodules/probot
 git fetch
-git checkout $HEAD
+git checkout "$HEAD"
 cd ../..
 
 # Commit and publish
 git add .
 git commit -m "Update docs for v$VERSION"
-git tag -f v$VERSION
+git tag -f "v$VERSION"
```

Comment:
After migration to semantic-release, the [`VERSION`](https://github.com/probot/probot/pull/1100/files#diff-0bd489d11e8d800c23c714c257307e66R7) variable will always be `0.0.0-development`.

We could use `npm show probot version` to get the latest published version, which was published by the previous stage.

Reply:
This issue should be resolved with a2f6ecfa501a6c7ef1c01305437764849784bfb8